modules/templates/template_proxy_request.rb

```diff
@@ -29,6 +37,14 @@ def get_template(kind, env, params)
       res.body
     end
 
+    def get_template(kind, env, params)
```

Comment:
Since it's not about just tempaltes anymore, what about calling the methods just `get` and `post`?